### ISSUE TITLE
Can I display username after receiving new message?

### ISSUE THREAD

**Issue by Znack** [vysheng/tg#517]
When new message comes telegram-cli display something like:
```
[19:15]  Sergey Cherepanov ««« Text of my message
```
Can I somehow display my, not username in this output
```
[19:15]  user#123456789 ««« Text of my message
```
Thanks.

**Comment by vysheng** [comment]
Is ```--disable-names``` is what you want? 

(it is in test branch. In master branch there was -I key for same purpose)

**Comment by Znack** [comment]
Unfortunately I'm not able to launch the current version and I use tag 1.0.5.1 that works pretty well :)
When I start current stable version it doesn't ask me for auth data and therefore I can't send and receive any message:
```
$ telegram-cli
Telegram-cli version 1.2.0, Copyright (C) 2013-2015 Vitaly Valtman
Telegram-cli comes with ABSOLUTELY NO WARRANTY; for details type `show_license'.
This is free software, and you are welcome to redistribute it
under certain conditions; type `show_license' for details.
Telegram-cli uses libtgl version 1.2.0
I: config dir=[/home/tochium/.telegram-cli]
> 
```

**Comment by LukeLR** [comment]
Does .telegram-cli-folder already exist? The you should try deleting (or renaming to e.g. .telegram-cli_old) that folder

**Comment by Znack** [comment]
@LukeLR do you mean .telegram-cli in my home directiory? 
I tried to remove it a number of times, it doesn't help :)

**Comment by LukeLR** [comment]
Yep, that's what I meant, I thought that it could possibly created by another version of telegram and that it can't read that files

**Comment by Znack** [comment]
Yeah, it was my first though too :)
But thanks for help anyway.
